webpack.config.js

```diff
@@ -1,14 +1,43 @@
-{
-  module.exports = {
-    entry: "./lib/index.js",
-    output: {
-      path: __dirname,
-      filename: "./dist/react-dropzone.js",
-      libraryTarget: "var",
-      library: "Dropzone"
+var webpack = require('webpack');
+var path = require('path');
+
+module.exports = {
+	entry: "./src/index.js",
+    devtool: "source-map",
+	output: {
+		path: __dirname,
+		filename: "./dist/react-dropzone.js",
+		libraryTarget: "umd",
+		library: "Dropzone"
+	},
+    module: {
+        loaders: [
+            {
+                include: [
+                    path.resolve(__dirname,"src"),
+                ],
+                test: /\.js$/,
+                loader: 'babel-loader',
+                query: {
+                    presets: ['es2015','react','stage-0'],
+                    plugins: ['add-module-exports']
+                }
+            }
+        ]  
     },
-    externals: {
-      "react": "React"
-    }
-  }
+    resolve: {
+        // Can require('file') instead of require('file.js') etc.
+        extensions: ['','.js','.json']
+    },
+	externals: {
+		"react": "React"
+	},
+    plugins: [
+        new webpack.optimize.UglifyJsPlugin({
+            compress: {
+                // does this actually do anything?
```

Comment:
It prevent UglifyJS from outputting a lot of warnings.

Also, we might want `screw_ie8` option be set to true here (check the uglifyJS options to make sure the name is correct)